content/collections/guides_and_surveys/en/sdk.md

```diff
@@ -49,14 +49,15 @@ amplitude.add(engagementPlugin());
 
 ### Third-party analytics provider
 
-Using the Guides and Surveys standalone SDK with another analytics provider requires extra configuration to help map properties to Amplitude.
+If you don't use the Amplitude Analytics [Browser SDK 2](https://amplitude.com/docs/sdks/analytics/browser/browser-sdk-2), you can still use Guides and Surveys but you need to configure the SDK to work with your third-party analytics provider. First, add the SDK to your project using the script tag, or via npm or Yarn as outlined above.
+But, instead of calling `amplitude.add(window.engagement.plugin())`, you'll need to call `init` and `boot`.
 
-{{partial:collapse name="Usign NPM or Yarn"}}
```

Comment:
Since `init` can now be safely called multiple times I would like to remove this as a special case and just document it generally. At some point, it would be great to not do `init` automatically with the script so people can properly use the arguments of  the method.